components/ResultsPage/Modal.tsx

```diff
@@ -0,0 +1,60 @@
+import { getTranslations } from '../../i18n/api'
+import { WebTranslations } from '../../i18n/web'
+import { MaritalStatus } from '../../utils/api/definitions/enums'
+import { useTranslation } from '../Hooks'
+
```

Comment:
Broken for me: 

<img width="1036" alt="image" src="https://github.com/DTS-STN/eligibility-estimator/assets/17856371/f4e1dce1-02ec-43ec-9219-c9cd7eccccae">